app/javascript/controllers/dropzone_controller.js

```diff
@@ -72,12 +72,15 @@ export default class extends Controller {
   }
 
   get addRemoveLinks() {
-    return this.data.get("addRemoveLinks") || true;
+    return this.data.get("addRemoveLinks") || false;
   }
 }
 
 class DirectUploadController {
+  // source is the input element
+  // file is the File object from dropzone
   constructor(source, file) {
+    this.count = Math.floor(Math.random() * 1_000_000_000)
```

Comment:
What is this count for?

Reply:
Need a unique identifier when you submit rails nested attributes.  It's the same sort of thing that's going on here (but timestamp doesn't have enough entropy in this case): https://github.com/sul-dlss/happy-heron/blob/main/app/javascript/controllers/nested_form_controller.js#L8